Demo/Sources/Views/PlaybackSlider.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  Copyright (c) SRG SSR. All rights reserved.
+//
+//  License information is available from the LICENSE file.
+//
+
+import Player
+import SwiftUI
+
+#if os(iOS)
+struct PlaybackSlider: View {
+    let progressTracker: ProgressTracker
+    let minimumValueText: String?
+    let maximumValueText: String?
+
+    @Binding private var value: Float
+    @State private var valueWidth: CGFloat = 0
+    @State private var previousValueWidth: CGFloat = 0
+
+    @Binding private var buffer: Float
+    @State private var bufferWidth: CGFloat = 0
+
+    @State private var isDragging = false {
+        didSet {
+            progressTracker.isInteracting = isDragging
+        }
+    }
```

Comment:
Probably superfluous, we can probably mutate `progressTracker.isInteracting` directly. The `progressTracker` should likely be made an `@ObservedObject` in the process.

Reply:
Yep! However, it seems to work without putting the `progressTracker` as `@ObservedObject`.